Source/Entities/Validation/CommonRules.cst

```diff
@@ -857,6 +888,73 @@ namespace <%=NameSpace%>.Validation
         }
 
         #endregion		
+
+        /// <summary>
+        /// Assigns an error string, and calls a handler if it exists
+        /// </summary>
+        /// <param name="ruleType"></param>
+        /// <param name="target"></param>
+        /// <param name="e"></param>
+        /// <param name="err"></param>
+        private static void AssignError(CommonRuleType ruleType, object target, ValidationRuleArgs e, string err)

```

Comment:
This method was created to provide a central place for event handling.  I am still running off the event handling I submitted for a pull request last time though, and am only about half way through writing a subscription based event handling system.  So for now I have removed the event code from this method.